tests/src/integration/webaction/webaction_test.go

```diff
@@ -0,0 +1,32 @@
+// +build integration
+
+package tests
+
+import (
+	"github.com/openwhisk/openwhisk-wskdeploy/cmdImp"
+	"github.com/stretchr/testify/assert"
+	"os"
+	"testing"
+)
+
+// TODO: write the integration against openwhisk
+func TestWebAction(t *testing.T) {
```

Comment:
Please be advised that we have changed the integration tests to run against binaries.